eng/pipelines/templates/stages/archetype-sdk-client.yml

```diff
@@ -3,7 +3,7 @@ resources:
     - repository: azure-sdk-build-tools
       type: git
       name: internal/azure-sdk-build-tools
-      ref: refs/tags/azure-sdk-build-tools_20200706.4
+      ref: refs/heads/OrganizePublishedArtifacts
```

Comment:
I assume this will be updated before merging.

Reply:
Yes, I will update that.

Reply:
Didn't you merge your azure-sdk-build-tools PR?

Reply:
Not Yet.